src/Log/InternalLogger.php

```diff
@@ -94,10 +94,6 @@ public function log($level, $message, array $context = [])
 
     private function shouldLog($currentLevel, array $context)
     {
-        if ($this->globalLevel == 0 && !$this->hasAnythingToIgnore($context)) {
-            return true;
-        }
-
         if ($currentLevel >= $this->globalLevel && !$this->hasAnythingToIgnore($context)) {
```

Comment:
Vagy félreértem, hogy mi a célja ennek a `shouldLog()` függvények, vagy nem értem, hogy ez az if() miért jó.
Légyszi mondd el, hogy mi a cél.

Reply:
Ez a metódus dönti el, hogy a soron következő log az SDKn beállitott LogLevel alapján logolandó-e.
Pl: ha az SDKn info a log level, akkor a `.debug()`-al irt logok nem logolódnak.

Reply:
mert azt az ágat fedi le amikor mehet a logolás, ha az SDK-ban beállitott log levelnél az aktuálisan logolandó level nagyobb és nem tartalmaz olyan exception-t sem ami miatt ignorálni kéne.

Szóval ha az SDK szintű level (a `$globalLevel`) mondjuk `WARNING` aminek az erteke `40`, akkor a `20`-as `INFO` (a `$currentLevel`) az nem fog logolodni mert itt a feltetel nem teljesul.